README.md

```diff
@@ -1,5 +1,9 @@
 # Angular Material 5.x to 6.0 updater tool
 
+**Outdated**: This repository won't be updated anymore. Future major releases for Angular Material will also have an upgrade tool that can be used in **Angular CLI** projects by running **`ng update @angular/material`**
```

Comment:
Is there any support for running the update schematic in a non-Angular CLI project via the schematic CLI?

Reply:
Good question. I'm pretty sure that there is no easy way to get migration schematic run without the CLI *now*. The update schematic is not part of the public schematic collection.

I will explore exposing the schematic as part of the public collection once we've updated the tool to work for Angular v7 (and maybe document).